boards/vcc-gnd/yd_esp32/doc/index.rst

```diff
@@ -233,7 +233,7 @@ RGB LED
 =======
```

Comment:
There is a typo in the commit title: led_2812 instead of led_ws2812

Reply:
Oops!

Fixed it.